packages/apollo-engine-reporting/src/plugin.ts

```diff
@@ -209,9 +257,9 @@ export const plugin = <TContext>(
         didEncounterErrors(requestContext) {
           // Search above for a comment about "didResolveSource" to see which
           // of the pre-source-resolution errors we are intentionally avoiding.
-          if (!didResolveSource) return;
+          if (!didResolveSource || endDone) return;
```

Comment:
This line ties into a bug @abernix is fixing already, I think, as reported by Danielle

Reply:
I think `didEncounterErrors` can still be called after `didResolveOperation`, I'll rebease over @abernix pr.

Reply:
Oh right I forgot that endDone here actually means "either the request has ended, or we decided we are not tracing". Maybe we should rename it?  What if we flip its value, and call it `traceActive` or something?